src/lib/styles/themes/light.scss

```diff
@@ -138,7 +168,6 @@
 
     /* Light/Text/Text */
     --text-color: var(--blue-900);
-    // TODO-colors: Remove application of rgbs
```

Comment:
Why did you removed these TODOs? I don't see anything about it in the PR descriptions.

Reply:
I initially removed them because the comments were inconsistent, and this was already addressed in another branch. But you’re right; removing them here is confusing. Restored.